src/Tribe/Assets.php

```diff
@@ -100,7 +95,7 @@ public function register_in_wp( $assets = null ) {
 				if ( did_action( $action ) > 0 ) {
 					$this->enqueue();
 				} else {
-					add_action( $action, [ $this, 'enqueue' ], $asset->priority );
+					add_action( $action, [ $this, 'enqueue' ], $asset->priority, 0 );
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: 5773c86e1e59a758cf3d0f7cf396c31e</sub>